bears/crystal/CrystalLintBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.ShardRequirement import (
+    ShardRequirement)
+
+
+@linter(executable='ameba',
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+)\n'
+                     r'.+31m(?P<origin>[A-Za-z]+): (?P<message>.*)')
```

Comment:
31m is ansi stuff?

we have a cleaner for that : `strip_ansi`

Reply:
Yes.
> 31m is ansi stuff?

Okay, I will look into it.
> we have a cleaner for that : strip_ansi